model/src/CMakeLists.txt

```diff
@@ -157,9 +157,24 @@ endif()
 
 # Handle PDLIB, SCRIP, SCRIPNC build files directly instead of through configuration file
 if("PDLIB" IN_LIST switches)
+  if("SCOTCH" IN_LIST switches)
+  find_package(PTSCOTCHparmetis REQUIRED)
+  find_package(PTSCOTCH REQUIRED)
+  find_package(PTSCOTCHerr REQUIRED)
+  find_package(SCOTCHerr REQUIRED)
+  find_package(SCOTCH REQUIRED)
+  target_sources(ww3_lib PRIVATE ${pdlib_src})
+  target_link_libraries(PTSCOTCHparmetis::PTSCOTCHparmetis INTERFACE PTSCOTCHerr::PTSCOTCHerr SCOTCHerr::SCOTCHerr PTSCOTCH::PTSCOTCH SCOTCH::SCOTCH)
+  target_link_libraries(ww3_lib PUBLIC PTSCOTCHparmetis::PTSCOTCHparmetis SCOTCHerr::SCOTCHerr)
+ endif()
+endif()
+
+if("PDLIB" IN_LIST switches)
```

Comment:
Why not have this in the same "if("PDLIB" IN_LIST switches)" if statement as above?

Reply:
Done